frontend-react/package.json

```diff
@@ -86,7 +86,8 @@
             "[/\\\\]node_modules[/\\\\](?!(react-markdown|vfile|vfile-message|markdown-table|unist-.*|unified|bail|is-plain-obj|trough|remark-.*|rehype-.*|html-void-elements|hast-util-.*|zwitch|hast-to-hyperscript|hastscript|web-namespaces|mdast-util-.*|escape-string-regexp|micromark.*|decode-named-character-reference|character-entities|property-information|hast-util-whitespace|space-separated-tokens|comma-separated-tokens|pretty-bytes|ccount|mdast-util-gfm|gemoji)).+\\.(js|jsx|mjs|cjs|ts|tsx)$"
         ],
         "moduleNameMapper": {
-            "axios": "axios/dist/node/axios.cjs"
+            "axios": "axios/dist/node/axios.cjs",
+            "\\.(css|less)$": "<rootDir>/src/__mocks__/styleMock.js"
```

Comment:
Let's add `scss` in here as well -- we'll probably need it!

Also, TINY POINT, but I'm assuming we can use this as a general catch-all for any static asset mock (like .jpg, .png, .md, et cetera) so we might want to just name this something like `staticAssetMock.js` instead.  Not a huge deal, though 🤷